---
device: multi
input_channel: 6
obsv_len: 5
out_layer: poselu
pred_len: 20
---
batch_size: 20
checkpoint_dir: model/wsd_poselu_enll
early_stopping: 0
epoch: 20
learning_rate: 0.001
weight_regularization: 0.0001
---
dataset_label_path: data/warehouse_sim_dataset/train/all_data.csv
dataset_name: warehouse_sim_dataset
dataset_path: data/warehouse_sim_dataset/train
model_path: model/wsd_poselu_enll
